.cirrus.yml

```diff
@@ -0,0 +1,54 @@
+task:
+  trigger_type: manual
+  name: E2E/Integration Tests
+  timeout_in: 45
+  container:
+    image: ubuntu:21.04
+    kvm: true # required for E2E/Integrations tests
+    cpu: 6
+    memory: 12G
+  env:
+    DEBIAN_FRONTEND: noninteractive
+    CI: true
+    CIRRUS_CLONE_DEPTH: 0 # Required to get app version
+    FORCE_COLOR: 1 # force terminal color
+    PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD: 1 # disable auto-browser download
+    KEYRING: /usr/share/keyrings/nodesource.gpg
+    NODE_VERSION: node_14.x # set node version e.g: "node_16.x" for Node 16 LTS
+
+  # Setting up a non-root user
+  prepare_user_script:
+    - groupadd -f -g $(stat -c '%g' /dev/kvm) kvm
+    - useradd -m -G kvm ranchertest
+    - tar c --owner=ranchertest.ranchertest . | tar x --directory=/home/ranchertest
```

Comment:
`tar` has separate `--owner` and `--group` options; also, I'm not sure what `ranchertest.ranchertest` is supposed to be… `chown` (on the next line) accepts `owner:group` (using a colon instead of a dot) instead.  Is it taking the second `ranchertest` as a machine name?

Reply:
Yep, I spend a bunch of time debugging it and on Ubuntu 21.04 groupadd owner:group has changed to `owner.group`
Moving back to Ubuntu LTS, it will be `owner:group`

Reply:
Sorted out the groupadd and tar errors. Passing only the user on tar step fixed the group id error. Everything seems fine now.
https://cirrus-ci.com/task/5241162284924928